tests/test_type_inference/test_function_annotation.py

```diff
@@ -0,0 +1,137 @@
+import astroid
+from typing import Any
+from nose.tools import eq_
+from python_ta.transforms.type_inference_visitor import TypeInferer
+from python_ta.typecheck.base import TypeFail
+import tests.custom_hypothesis_support as cs
+
+
+def test_single_annotation_int():
+    src = '''
```

Comment:
In general use double-quotes for triplequotes and single-quotes for single-line string literals.